app/Jobs/ImportRockTheVoteRecord.php

```diff
@@ -211,6 +220,15 @@ public function handle(Rogue $rogue)
             $this->sendUserPasswordResetIfSubscribed($user);
         }
 
+        $rockTheVoteRecord = RockTheVoteRecord::firstOrCreate([
+            'import_file_id' => $this->import_file_id,
+            'finish_with_state' => $this->record->rtv_finish_with_state,
+            'started_registration' => $this->record->rtv_started_registration,
+            'status' => $this->record->rtv_status,
+            'tracking_source' => $this->record->rtv_tracking_source,
+            'user_id' => $user->id,
+        ]);
```

Comment:
This could be a good place to use a static initializer method on `RockTheVoteRecord`:

```php
static function createFromRecord(ParsedRockTheVoteRecord $record, NorthstarUser $user, $fileId)
{
    return static::firstOrCreate([
       'import_file_id' => $this->import_file_id,
       'finish_with_state' => $this->record->rtv_finish_with_state,
       'started_registration' => $this->record->rtv_started_registration,
       'status' => $this->record->rtv_status,
       'tracking_source' => $this->record->rtv_tracking_source,
       'user_id' => $user->id,
    ]);
}
```

Then in here:

```php
$rockTheVoteRecord = RockTheVoteRecord::createFromRecord($this->record, $user, $this->import_file_id);
```